core-java/src/main/java/com/github/wasiqb/boyka/manager/ApiManager.java

```diff
@@ -157,10 +158,11 @@ private String getRequestBody (final RequestBody body) {
     private ApiResponse getResponse (final String path) {
         LOGGER.traceEntry ("Parameters: {}", path);
         try {
-            this.response = parseResponse (this.client.newCall (this.request.url (getUrl (path))
-                    .build ())
-                .execute ());
-            logRequest ();
+            Response response = this.client.newCall (this.request.url (getUrl (path))
```

Comment:
I don't see any reason to create a local variable here for `response`. Can you revert it?